modules/_node-scripts/configuration/getProjectEntryPoints.mjs

```diff
@@ -14,10 +14,13 @@ import projectScopeRequire from '../util/projectScopeRequire.mjs';
  * {
  *   main: 'src/main/resources/META-INF/resources/index.js',
  *   submodules: {
- *   	foo: 'src/main/resources/META-INF/resources/foo.js'
+ *     foo: 'src/main/resources/META-INF/resources/foo.js'
  *   },
  *   typescript: {
- *   	main: 'src/main/resources/META-INF/resources/index.d.ts'
+ *     main: 'src/main/resources/META-INF/resources/index.d.ts'
+ *     submodules: {
+ *       foo: 'src/main/resources/META-INF/resources/foo.d.ts'
+ *     }
```

Comment:
nit: these paths should start with `./`

Reply:
see https://github.com/liferay-frontend/liferay-portal/pull/4742#discussion_r1961558176